src/EA.Weee.Integration.Tests/Builders/EvidenceNoteDbSetup.cs

```diff
@@ -63,5 +63,13 @@ public EvidenceNoteDbSetup WithTonnages(List<NoteTonnage> tonnages)
             DefaultNote(tonnages);
             return this;
         }
+
+        //public EvidenceNoteDbSetup UpdateTonnages(List<NoteTonnage> tonnages)
```

Comment:
Is this ever needed again? Would it be neater to just remove?

Reply:
Should be gone now...